CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - Major: Added "Channel Filters". See https://wiki.chatterino.com/Filters/ for how they work or how to configure them. (#1748, #2083, #2090, #2200)
 - Major: Added Streamer Mode configuration (under `Settings -> General`), where you can select which features of Chatterino should behave differently when you are in Streamer Mode. (#2001)
+- Minor: Improve search popup window titles (#2268)
```

Comment:
nit-pick: Could use perfective verb instead of infinitive to keep consistency (which I know is broken in some places, but most entries use perfective verbs anyway).
```suggestion
- Minor: Improved search popup window titles (#2268)
```